src/Codeception/Subscriber/Console.php

```diff
@@ -536,7 +536,9 @@ protected function detectWidth()
             }
         } elseif ($this->isWin() && (php_sapi_name() === "cli")) {
             exec('mode con', $output);
-            preg_match('/^ +.* +(\d+)$/', $output[4], $matches);
+            if (isset($output[4])) {
```

Comment:
The `PHP Notice` occurs in the `preg_match` line when `$output[4]` is _not_ set.  The patch just makes sure that it _is_ set before trying to use it.  It does solve the `undefined offset` notice that I was seeing on my Windows box.